### ISSUE TITLE
PEP 541 Request: vapi-runtime

### ISSUE THREAD

**Issue by aagrawal3** [pypi/support#3214]
### Project to be claimed

`vapi-runtime`: https://pypi.org/project/vapi-runtime

### Your PyPI username

`aagrawal3`: https://pypi.org/user/aagrawal3

### Reasons for the request

I could not find a project with same name on PyPI but getting following error:

> INFO     Response from https://upload.pypi.org/legacy/:
         400 The name 'vapi-runtime' isn't allowed. See https://pypi.org/help/#project-name for more information.
INFO     <html>
          <head>
           <title>400 The name 'vapi-runtime' isn't allowed. See https://pypi.org/help/#project-name for more information.</title>
          </head>
          <body>
           <h1>400 The name 'vapi-runtime' isn't allowed. See https://pypi.org/help/#project-name for more information.</h1>
           The server could not comply with the request since it is either malformed or otherwise incorrect.<br/><br/>
         The name &#x27;vapi-runtime&#x27; isn&#x27;t allowed. See https://pypi.org/help/#project-name for more information.
          </body>
         </html>
ERROR    HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
         The name 'vapi-runtime' isn't allowed. See https://pypi.org/help/#project-name for more information.

### Maintenance or replacement?

Maintenance

### Source code repositories URLs

https://github.com/vmware/vsphere-automation-sdk-python/tree/master/lib/src/vapi-runtime

### Contact and additional research

Tried eliminating possibilities for the issue as per guidelines available in 
https://pypi.org/help/#project-name

There is a project with same name on TEST pypi published by an ex-colleague, not sure if this could be the reason
https://test.pypi.org/project/vapi-runtime/

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by aagrawal3** [comment]
I found some history around it. There are 3 projects under following ticket
https://github.com/pypi/support/issues/105

One of them "vapi-client-bindings" was reclaimed through https://github.com/pypi/support/issues/946
> Hello, the project name vapi-client-bindings was placed in the prohibited name list by @pypa/warehouse-admins, I don't see a problem making it available again but I don't have the permissions to do so.

The other two (vapi-runtime and vapi-common-client) I think are still in "Prohibited list"
Tagging @di @yeraydiazdiaz for help

**Comment by aagrawal3** [comment]
@pypa/warehouse-admins

**Comment by aagrawal3** [comment]
Hi,
Any update on this?

**Comment by aagrawal3** [comment]
Following up on this request.

**Comment by aagrawal3** [comment]
Following up. Please take a look at https://github.com/pypi/support/issues/3214#issuecomment-1736980461

**Comment by ShadowJonathan** [comment]
@aagrawal3 the URL https://pypi.org/project/vapi-runtime gives back a 404

**Comment by aagrawal3** [comment]
> @aagrawal3 the URL https://pypi.org/project/vapi-common-client gives back a 404, maybe try again? Does it work?

Tried again, still 400
We suspect it is because this project name is in prohibited list. Please refer to https://github.com/pypi/support/issues/3214#issuecomment-1736980461

**Comment by mariolenz** [comment]
Any news on this?

**Comment by mariolenz** [comment]
@alex @cmaureir @di @dstufft @ewdurbin @FFY00 @miketheman @webknjaz @woodruffw Any news on this, or any help you could provide?

**Comment by woodruffw** [comment]
@mariolenz This is a volunteer-run support channel, and most (all?) of the people you've pinged are probably currently celebrating Christmas or otherwise not online. Please be patient, and note that there may be a significant delay to other requests being reviewed ahead of yours.

**Comment by woodruffw** [comment]
(Also note: several of the people you've tagged, including myself, have no ability to help with this request. Please, *please*, do not assume that all PyPA members are unilaterally capable of responding to your support requests.)

**Comment by mariolenz** [comment]
> Also note: several of the people you've tagged, including myself, have no ability to help with this request. Please, _please_, do not assume that all PyPA members are unilaterally capable of responding to your support requests.

Sorry, it looks like I've misunderstood this repo. Since you're mentioned [here](https://github.com/orgs/pypi/people) I thought you're the persons to ping about this. Please forgive me if I was was wrong there.

> This is a volunteer-run support channel, and most (all?) of the people you've pinged are probably currently celebrating Christmas or otherwise not online.

Not to put too fine a point on this, but the issue has been opened end of September... three months before Christmas ;-)

**Comment by woodruffw** [comment]
> Sorry, it looks like I've misunderstood this repo. Since you're mentioned [here](https://github.com/orgs/pypi/people) I thought you're the persons to ping about this. Please forgive me if I was was wrong there.

Yes, only a subset of the PyPI and PyPA maintainers have the ability to service these requests. They receive notifications on these issues and are aware of them, but there's a significant backlog due to it being entirely volunteer time. 

(That's to say: understandable mistake to make, but there's no need to ping public members! Not everybody is a public member, and those who need to see it will see it regardless 🙂)

> Not to put too fine a point on this, but the issue has been opened end of September... three months before Christmas ;-)

Yes, but this isn't a good reason to ping on a day when people are unlikely to be active (or indiscriminately, although this isn't your fault per above). There are dozens of other PEP 541 requests open before September; for better or worse, this backlog will take a while to clear.

(I acknowledge that this is an unsatisfying answer. Unfortunately, it's the reality of the ratio between PyPI's support volume and the amount of maintainer time available.)

**Comment by webknjaz** [comment]
@mariolenz echoing William's sentiment — please avoid subscribing a bunch of people to issues. I don't have any PyPI powers either. _(clicking "unsubscribe" now)_

**Comment by Thespi-Brain** [comment]
Hello @aagrawal3,

This name cannot be released as it poses a security risk. Please choose another name, thanks.
